mars/tensor/tests/test_core_execute.py

```diff
@@ -48,11 +48,11 @@ def testArrayFunction(self):
         a = ones((10, 20), chunk_size=5)
 
         # test sum
-        self.assertEqual(np.sum(a).execute(), 200)
+        self.assertEqual(np.sum(a).execute(session=self.session), 200)
```

Comment:
Why need pass the session?

Reply:
Yes, it's a bug. isclose don't support jax (becuase it don't use any numpy-bin function). So I remove its jax support